configs/x86_64/host.config

```diff
@@ -5968,3 +5968,13 @@ CONFIG_DEBUG_INFO_REDUCED=n
 CONFIG_DEBUG_INFO_NONE=n
 CONFIG_DEBUG_INFO_BTF=y
 CONFIG_DEBUG_INFO_BTF_MODULES=y
+
+# Hyper-V vsock
+CONFIG_HYPERV_VSOCKETS=m
+
+# Needed for virtio_mem
+CONFIG_VIRTIO_MEM=m
+CONFIG_MEMORY_HOTPLUG=y
+CONFIG_MEMORY_HOTREMOVE=y
+CONFIG_ARCH_ENABLE_MEMORY_HOTREMOVE=y
+CONFIG_HAVE_BOOTMEM_INFO_NODE=y
```

Comment:
Ideally we should enable this for arm64 as well to keep these consistent.

Can you just throw it in there?

Reply:
Done!